upgrade/1.2/Stage_0_Prerequisites.md

```diff
@@ -171,6 +171,14 @@ Run check script:
    ncn-m001# /usr/share/doc/csm/upgrade/1.2/scripts/upgrade/prerequisites.sh --csm-version [CSM_RELEASE] --tarball-file [PATH_TO_CSM_TARBALL_FILE]
    ```
 
+**NOTE** During the UPDATE\_SSH\_KEYS phase, you will need to respond to interactive prompts for connecting to each NCN. Respond with yes to these prompts. Here is an example:
+
+```bash
```

Comment:
This should probably change to `text` or `console` since this isn't technically bash and it shouldn't get code-coloring/highlighting like a bash script would.